app/helpers/helpers.rb

```diff
@@ -0,0 +1,42 @@
+helpers do
```

Comment:
Helpers are intended to provide methods to your views, not your controller.  These should all just be methods in `server.rb`